src/Icons/doc/index.rst

```diff
@@ -19,10 +19,60 @@ Installation
 
     $ composer require symfony/ux-icons
 
-Icons?
+Icons
 ------
 
-No icons are provided by this package but there are several ways to include and render icons.
+The ``symfony/ux-icons`` package aspires to offer simple and easy-to-use ways to
```

Comment:
I wouldn't say `aspires to offer` because it looks like a wish instead of a reality.